token_test.go

```diff
@@ -0,0 +1,77 @@
+package jwt_test
+
+import (
+	"testing"
+
+	"github.com/golang-jwt/jwt"
```

Comment:
Please rebase to main and updater the import path to `github.com/golang-jwt/jwt/v4` for this test file to work